controllers/user.controller.js

```diff
@@ -88,6 +88,12 @@ exports.register = async (req, res) => {
       msg: "Please enter all fields",
     });
   }
+  if (personal_info.password !== personal_info.confirmPassword){
```

Comment:
I think we can keep this check on frontend only. After comparing the two, frontend will send one final password.